### ISSUE TITLE
Useful mac-oriented ZSH installer with lots of useful code you might want to borrow

### ISSUE THREAD

**Issue by beekman** [mhulse/dotfizzles#205]
https://github.com/atomantic/dotfiles/blob/master/install.sh#L275-L1038 shows everything that it does when you run the installer. Definitely worth skimming for useful stuff to incorporate into dotfizzles.

**Comment by mhulse** [comment]
Nice find! Looks like it installs homebrew deps too? That's pretty interesting. I never considered having an install script help with system deps (dotfizzles assumes there's a few, and/or I suggest a few upgrades/tweaks in terms of system deps).

Anyway, thanks @beekman for the linkage/tip! I really need to clean this repo up ... SOON!!!!!! 🚀

**Comment by beekman** [comment]
Yep, this is the same one I installed on both of my MBP's-- it does all SORTS of things. From the Readme: "I will update your MacOS machine with Better™ system defaults, preferences, software configuration and even auto-install some handy development tools and apps that my developer friends find helpful. You don't need to install or configure anything upfront! This works with a brand-new machine from the factory as well as an existing machine that you've been working with for years."
